src/nodes.coffee

```diff
@@ -806,7 +815,10 @@ exports.Block = class Block extends Base
     body = []
     for expression in @expressions
       expressionAst = expression.ast o
-      if expression instanceof Directive
+      # Ignore generated PassthroughLiteral
```

Comment:
Since there are "dummy" generated `PassthroughLiteral` nodes that just exist to get comments into the nodes tree, there are a couple places here where we have to explicitly ignore them for purposes of AST generation